src/components/ConfigEditor/ConfigEditor.tsx

```diff
@@ -23,10 +24,25 @@ import { lastValueFrom } from 'rxjs';
 import { parseRepositoriesResponse } from 'utils/utils';
 import { DefaultRepository } from './DefaultRepository';
 import { Divider } from './Divider';
+import { css } from '@emotion/css';
+
+const getStyles = (theme: GrafanaTheme2) => ({
+  toggle: css`
+    margin-top: 7px;
+    margin-left: 5px;
+  `,
+  infoText: css`
+    padding-bottom: ${theme.v1.spacing.md};
+    color: ${theme.v1.colors.textWeak};
+  `,
+});
```

Comment:
im not a fan of styles in .tsx and modifying styling. does it look bad with default styling?

Reply:
I used the same styles from another datasource for consistency. I can check to see what it looks like without.